### PR TITLE
[IMP] marketing automation: updated workflow activities for 16

### PR DESCRIPTION
**Pull request by ksc-odoo**
Project Task: https://www.odoo.com/web#id=3425916&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 1]
```diff
@@ -1,80 +1,251 @@
...
-activity and return to the marketing automation campaign form, :guilabel:`Save & New` to save the
-activity and immediately create another one in a fresh :guilabel:`Create Activities` pop-up window,
-or :guilabel:`Discard` to delete the activity and return to the marketing automation campaign form.
+   :alt: A workflow activity template pop-up window in Odoo Marketing Automation.
+
+First, enter a name for the activity in the :guilabel:`Activity Name` field, and proceed to
+configure the following options. Once ready, either click: :guilabel:`Save & Close` (to save the
+activity and close the pop-up form), :guilabel:`Save & New` (to save the activity and instantly
+create an additional activity on a fresh :guilabel:`Create Activities` pop-up form), or
+:guilabel:`Discard` to delete the activity.
```
Comment (brse-odoo) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:27]:
```suggestion
configure the following options. Once ready, either click: :guilabel:`Save & Close` to save the
activity and close the pop-up form, :guilabel:`Save & New` to save the activity and instantly
create an additional activity on a fresh :guilabel:`Create Activities` pop-up form, or
:guilabel:`Discard` to delete the activity.
```

Comment (brse-odoo) [reply content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:27]: I think it flows fine without parentheses, even as a longer sentence

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 2]
```diff
@@ -1,80 +1,251 @@
...
+
+Activity types
+--------------
+
+Then, select the :guilabel:`Activity Type`. Choose between :guilabel:`Email`, :guilabel:`Server
+Action` (an internal action within the database), or :guilabel:`SMS`.
```
Comment (brse-odoo) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:33]: Is there a doc about server actions we could link to here, just so the curious-minded could read more about them? Not totally necessary, but would be a nice-to-have

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 3]
```diff
@@ -1,80 +1,251 @@
...
+
+.. image:: workflow_activities/email-activity-create-edit.png
+   :align: center
+   :alt: The create and edit email drop-down option on create activities pop-up window.
+
+In that pop-up window, proceed to create and configure the new email template.
+
+.. image:: workflow_activities/create-marketing-template-popup-window.png
+   :align: center
+   :alt: The create marketing template email pop-up window looks like in Odoo Marketing Automation.
```
Comment (brse-odoo) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:57]:
```suggestion
   :alt: The create marketing template email pop-up window in Odoo Marketing Automation.
```

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 4]
```diff
@@ -1,80 +1,251 @@
...
+
+Server action activity
+~~~~~~~~~~~~~~~~~~~~~~
+
+If :guilabel:`Server Action` is the :guilabel:`Activity Type`, the option to choose a specific
+:guilabel:`Server Action` (e.g. Message for sales person, Create Leads on Website clicks, etc.)
+becomes available.
+
+The option to create a new server action directly from the :guilabel:`Server Action` is also
+available. To do that, either type in the name of the new action, then click :guilabel:`Create and
```
Comment (brse-odoo) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:67]:
```suggestion
available. To do that, type in the name of the new action, then click :guilabel:`Create and
```

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 5]
```diff
@@ -1,80 +1,251 @@
...
+Edit...` from the drop-down menu.
+
+.. image:: workflow_activities/server-action-create-edit-option.png
+   :align: center
+   :alt: The create and edit option in the server action field on campaign detail form.
+
+Or, click the empty :guilabel:`Server Action` field to reveal a drop-down menu, and select
```
Comment (brse-odoo) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:74]:
```suggestion
Or, as an alternative, click the empty :guilabel:`Server Action` field to reveal a drop-down menu, and select
```

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 6]
```diff
@@ -1,80 +1,251 @@
...
+- :guilabel:`Mail:opened`: the activity will be triggered at the previously-configured time if the
+  sent mail in the workflow has been opened by the recipient.
+- :guilabel:`Mail:not opened`: the activity will be triggered at the previously-configured time if
+  the sent mail in the workflow has not been opened by the recipient.
+- :guilabel:`Mail: replied`: the activity will be triggered at the previously-configured time if the
+  sent mail in the workflow has been replied to by the recipient.
+- :guilabel:`Mail: not replied`: the activity will be triggered at the previously-configured time if
+  the sent mail in the workflow has not been replied to by the recipient.
+- :guilabel:`Mail: clicked`: the activity will be triggered at the previously-configured time if the
+  sent mail in the workflow has been clicked by the recipient, after its been opened.
```
Comment (brse-odoo) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:138]:
```suggestion
  sent mail in the workflow has been clicked by the recipient, after it's been opened.
```

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 7]
```diff
@@ -1,80 +1,251 @@
...
+- :guilabel:`Mail: not clicked`: the activity will be triggered at the previously-configured time if
+  the sent mail in the workflow has not been clicked by the recipient, after its been opened.
```
Comment (brse-odoo) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:140]:
```suggestion
  the sent mail in the workflow has not been clicked by the recipient, after it's been opened.
```

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 8]
```diff
@@ -1,80 +1,251 @@
...
+- :guilabel:`Mail: bounced`: the activity will be triggered at the previously-configured time if the
+  sent mail in the workflow has been bounced back for any reason.
+- :guilabel:`SMS: clicked`: the activity will be triggered at the previously-configured time if the
+  sent SMS in the workflow has been clicked by the recipient, after its been opened.
```
Comment (brse-odoo) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:144]:
```suggestion
  sent SMS in the workflow has been clicked by the recipient, after it's been opened.
```

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 9]
```diff
@@ -1,80 +1,251 @@
...
+- :guilabel:`SMS: not clicked`: the activity will be triggered at the previously-configured time if
+  the sent SMS in the workflow has not been clicked by the recipient, after its been opened.
```
Comment (brse-odoo) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:146]:
```suggestion
  the sent SMS in the workflow has not been clicked by the recipient, after it's been opened.
```

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst (hunk 6 above)
Comment (brse-odoo) [reply content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:138]: "it has" contracts to "it's"

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 10]
```diff
@@ -1,80 +1,251 @@
...
 
 Child activities
 ================
 
 There is also the option to add a *child activity* by clicking :guilabel:`Add child activity`,
 located at the bottom of each activity block in the :guilabel:`Workflow` section of a marketing
-automation form.
+campaign form.
 
 Child activities are sub-activities that are connected to (and triggered by) the activity above it,
```
Comment (brse-odoo) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:229]:
```suggestion
Child activities are sub-activities that are connected to and triggered by the activity above it,
```

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 11]
```diff
@@ -1,80 +1,251 @@
...
-which is also known as its *parent activity*.
+which is known as its *parent activity*. A child activity is easy to recognize as its slightly
```
Comment (brse-odoo) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:230]:
```suggestion
which is known as its *parent activity*. A child activity is easy to recognize, as its slightly
```

**Review by brse-odoo — COMMENTED** [review]
@ksc-odoo I've just finished my review of this doc, great work! Let me know if you have any questions about my comments/suggestions. Thanks!

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst (hunk 2 above)
Comment (ksc-odoo) [reply content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:33]: Oh, great idea. Unfortunately, I don't believe there is any specific doc about server actions. But I fully understand where something like that would come in handy in this instance. So, if anything like that *does* exist, I'll be sure to add it in here -- for sure! Thanks for the suggestion! 👍

**Comment by ksc-odoo** [comment]
Thanks for awesome suggestions, @brse-odoo -- I made all the changes/adjustments, so once you approve them -- I'll be able to move this on to the next stage of review. Thanks again! 👍

**Review by brse-odoo — APPROVED** [review]

**Comment by brse-odoo** [comment]
@ksc-odoo Just approved the changes, this one is ready to send to the next round! 👍

**Comment by ksc-odoo** [comment]
Thanks for the approval @brse-odoo ! This one's ready for another round of review from you, @meng-odoo -- whenever you get a chance! 👍

**Review by StraubCreative — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**